static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -196,10 +196,15 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities',  'A
 		}
 
 		// Export actions
-		if (!isContentRoot) {
-			items["sepexport"] = "";
+		if ($.inArray('viewHidden', metadata.permissions) !== -1) {
+			items["export"] = {name: "Export", items: {}}
+
+			if (!isContentRoot && metadata.type !== 'File' && $.inArray('viewHidden', metadata.permissions) != -1) {
+				items['export']['items']["exportCSV"] = {name: "Export CSV"};
+			}
+
 			if (metadata.type !== 'File' && $.inArray('viewHidden', metadata.permissions) != -1) {
-				items["exportCSV"] = {name : 'Export as CSV'};
+				items['export']['items']["exportMemberOrder"] = {name: "Export Member Order"};
```

Comment:
the "Member Order" export doesn't appear to do anything for me when i click on it here. For these menu items to perform an action, they have to have a corresponding callback in this section:
https://github.com/UNC-Libraries/box-c/blob/main/static/js/admin/src/ResultObjectActionMenu.js#L247
It looks like the behavior for triggering the export is defined here:
https://github.com/UNC-Libraries/box-c/blob/44b2d5a3f6a6031d0d865cc65cac4d72f2cf5340/static/js/admin/src/ExportMenu.js#L71-L76
So you will like need to add something like this in the ResultObjectActionMenu callbacks. The main difference is that in ExportMenu it is being given a list of ids to export, while here it is being given a single id.